ositrace/ositrace_shacl.ttl

```diff
@@ -0,0 +1,286 @@
+@prefix ositrace: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/ositrace/> .
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix georeference: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/georeference/> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+ositrace:OSITraceShape a sh:NodeShape ;
+    sh:property 
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:GeneralShape ;
+            sh:order 1 ;
+            sh:path ositrace:general ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:FormatShape ;
+            sh:order 2 ;
+            sh:path ositrace:format ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:ContentShape ;
+            sh:order 3 ;
+            sh:path ositrace:content ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:QualityShape ;
+            sh:order 4 ;
+            sh:path ositrace:quality ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:QuantityShape ;
+            sh:order 5 ;
+            sh:path ositrace:quantity ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node ositrace:DataSourceShape ;
+            sh:order 6 ;
+            sh:path ositrace:dataSource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node georeference:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path ositrace:georeference ] ;
+    sh:targetClass ositrace:OSITrace .
+
+ositrace:ContentShape a sh:NodeShape ;
+    sh:property [ skos:example "[motorway, residential]" ;
+            sh:datatype xsd:string ;
+            sh:description "Covered/used road types, defined over ODR element t_road_type, see ODR spec section 8.3"@en ;
+            sh:in ("Bicycle" "LowSpeed" "Motorway" "Pedestrian" "Rural" "Town" "TownArterial" "TownCollector" "TownExpressway" "TownLocal" "TownPlayStreet" "TownPrivate" "Unknown") ;
+            sh:message "Validation of roadTypes failed!"@en ;
+            sh:name "roadTypes"@en ;
+            sh:order 0 ;
+            sh:path ositrace:roadTypes ],
+        [ skos:example "shoulder, curb, ..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered lane types, see ODR spec section 9.5.3."@en ;
+            sh:in ("biking" "border" "connectingRamp" "curb" "driving" "entry" "exit" "median" "none" "offRamp" "onRamp" "parking" "restricted" "shoulder" "slipLane" "stop" "walking") ;
```

Comment:
I think it was just the same as in `hdmap` right? Otherwise, why is it mixed lower case here and the previous was starting with upper case?